Emby.Server.Implementations/Library/DefaultAuthenticationProvider.cs

```diff
@@ -47,7 +47,7 @@ public Task<ProviderAuthenticationResult> Authenticate(string username, string p
         {
             if (resolvedUser == null)
             {
-                throw new ArgumentNullException(nameof(resolvedUser));
+                throw new AuthenticationException($"Specified user does not exist.");
```

Comment:
For security reasons, I suggest to not be as specific : It avoids an attacker to find usernames.
"Invalid username or password" should be reused.

Reply:
`throw new AuthenticationException($"Invalid username or password.");`

Reply:
I also made a change so that the exception message is not sent as part of the response: https://github.com/jellyfin/jellyfin/pull/2861/files#diff-234173e74ae9494988401966dcd4f002R281